test/expected/plan_ordered_append-10.out

```diff
@@ -1170,11 +1170,11 @@ ORDER BY o1.time;
          ->  Index Scan Backward using _hyper_1_3_chunk_ordered_append_time_idx on _hyper_1_3_chunk o1_3 (actual rows=23043 loops=1)
                Index Cond: ("time" < 'Tue Feb 01 00:00:00 2000 PST'::timestamp with time zone)
    ->  Materialize (actual rows=220327 loops=1)
-         ->  Merge Append (actual rows=73443 loops=1)
-               Sort Key: o2."time"
-               ->  Index Scan Backward using _hyper_1_1_chunk_ordered_append_time_idx on _hyper_1_1_chunk o2 (actual rows=20160 loops=1)
-               ->  Index Scan Backward using _hyper_1_2_chunk_ordered_append_time_idx on _hyper_1_2_chunk o2_1 (actual rows=30240 loops=1)
-               ->  Index Scan Backward using _hyper_1_3_chunk_ordered_append_time_idx on _hyper_1_3_chunk o2_2 (actual rows=23043 loops=1)
+         ->  Custom Scan (ChunkAppend) (actual rows=73443 loops=1)
```

Comment:
is the comment above `no exclusion on joined table because its not ChunkAppend node` out of date?